analytic_engine/src/memtable/skiplist/iter.rs

```diff
@@ -168,8 +167,9 @@ impl<A: Arena<Stats = BasicStats> + Clone + Sync + Send> ColumnarIterImpl<A> {
 
         if num_rows > 0 {
             if let Some(deadline) = self.deadline {
-                if deadline.check_deadline() {
-                    return IterTimeout {}.fail();
+                let now = Instant::now();
+                if now.duration_since(deadline).is_zero() {
+                    return IterTimeout { now, deadline }.fail();
```

Comment:
Any problems in the previous implementation?

Reply:
In our env, some query return timeout immediately(set to `60s`), so add some info to debug why deadline happen so fast.